pkg/execution/execution.go

```diff
@@ -87,37 +88,24 @@ type Executor interface {
 		stackIndex int,
 	) (*state.DriverResponse, error)
 
-	// HandleResponse handles the response from running a step.
-	HandleResponse(
-		ctx context.Context,
-		id state.Identifier,
-		item queue.Item,
-		edge inngest.Edge,
-		resp *state.DriverResponse,
-	) error
-
-	// HandleGeneratorResponse handles all generator responses.
-	HandleGeneratorResponse(ctx context.Context, resp *state.DriverResponse, item queue.Item) error
-	// HandleGenerator handles an individual generator response returned from the SDK.
-	HandleGenerator(ctx context.Context, gen state.GeneratorOpcode, item queue.Item) error
```

Comment:
are these no longer needed because they're not required anymore and will always be using the OSS logic?
if so, we could just make the methods private.